assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/calico-host-protection/templates/host-endpoints.yaml

```diff
@@ -10,5 +10,5 @@ metadata:
     {{- end }}
 spec:
   interfaceName: {{ .interfaceName }}
-  node: {{ .name }}
+  node: {{ .nodeName }}
```

Comment:
Can we create an issue to add tests for yamls generation too? (Not sure if helm has some built-in support to make testing easier, but we can see when tackling the issue)

Reply:
What kind of tests?

Reply:
Test that generated yamls have the expected values

Reply:
I'm not familiar with any test framework for Helm and I'm not sure if it's worth investing time into that. IMO functional tests are more important first. E.g. to see if all nodes has associated HostEndpoint objects.

I was hoping to get this PR merged quickly, as it fixes security issue and then implement regression test for it.

Reply:
Sure, that is why I said the PR looks good :)